addons/csw/stringtable.xml

```diff
@@ -372,6 +400,7 @@
         </Key>
         <Key ID="STR_ACE_CSW_127x108_displayName">
             <English>[CSW] 12.7x108mm HMG Belt</English>
+            <German>[CSW] 12.7x108mm HMG Gurt</German>
```

Comment:
Prinzipiell gleiches Problem wie bei Depleted Uranium. In jedem Fall aber mit Bindestrich: "HMG-Gurt".

Gilt auch für die anderen Magazine, bis L458.